app/charges/controllers.py

```diff
@@ -188,14 +191,15 @@ def create_charge(user, user_data):
     charge.author = user.id
     charge.description = user_data.get("description", "")
     charge.committee = committee.id
-    charge.status = user_data.get("status", "") # TODO problem here
```

Comment:
Is there still a problem here? Should the status be set to in progress if status isn't provided?

Reply:
I think that comment is from another branch, but thats a good point i'll check it out.

Reply:
InProgress is the default selection for the html select element on the front end. So we are all good here